### PR TITLE
Enhance Go driver to read Db2 credentials form Evn for testing

### PR DESCRIPTION
**Pull request by vmathur12**
Enhance Go driver to read Db2 credentials form Environment variables for testing

### CODE REVIEW

File: testdata/main.go [hunk 1]
```diff
@@ -6,32 +6,122 @@ import (
...
+func GetConnectionInfoFromEnv() bool{
+	var databaseFound bool
+	var hostFound bool
+	var portFound bool
+	var uidFound bool
+	var pwdFound bool
+	var flag1 bool = false
+	database, databaseFound = os.LookupEnv("DB2_DATABASE")
+        if !databaseFound{
+		fmt.Println("Warning: Environment variable DB2_DATABASE is not set.")
```
Comment (bimalkjha) [inline testdata/main.go:66]: @vmathur12 Print warning only if env vars DB2_USER and DB2_PASSWD is not set. No need to set all connection string info from env vars only. Few can be read from config.json file too like SECURITY, SSLServerCertificate and other keywords including dbname, host, port.

File: testdata/main.go [hunk 2]
```diff
@@ -6,32 +6,122 @@ import (
...
+        }
+        jsonParser := json.NewDecoder(configFile)
+        err = jsonParser.Decode(&config)
+        return config, err
+}
+
+//Get connection information from config.json
+func GetConnectionInfoFromConfigFile() {
+       config, _:= LoadConfiguration("config.json")
+       host = config.Host
```
Comment (bimalkjha) [inline testdata/main.go:48]: Only read from config file if value is not set by env var. Env var setting will have more priority. Env var might have set port but not host. So, update code accordingly. Thanks.

**Review by bimalkjha — CHANGES_REQUESTED** [review]
Address the comments given in main.go file.
It would be good to move individual test functions in a test.go file and call them from main.go. I mean bifurcate the big main.go into smaller files.

File: testdata/main.go [hunk 3]
```diff
@@ -6,32 +6,122 @@ import (
...
+		if len(uid)==0 {
+		    fmt.Println("Warning: Environment variable DB2_USER is not set.")
+	        }
+	}
+
+        pwd, pwdFound = os.LookupEnv("DB2_PASSWD")
+        if !pwdFound{
+		pwd = config.Pwd
+
+		if len(pwd)==0 {
```
Comment (bimalkjha) [inline testdata/main.go:102]: @vmathur12 Remove this if check. We need to print warning for password if pwd is found in config file but env var DB2_USER is not set. Current code will not print warning if pwd is found in config file.

Comment (vmathur12) [reply testdata/main.go]: @bimalkjha  Removed if len(pwd)==0 check. Please review and merge it. Thanks

**Review by bimalkjha — APPROVED** [review]

**Pull request merged**